examples/plot_vector_testing.py

```diff
@@ -0,0 +1,205 @@
+# assert_collection_sorted_by_markersize().
+#
+# To check the geometry locations, you can call assert_points() and prove the
+# expected points data, which in this case is the point_gdf object.
+#
+# If the plot uses point markers that are sized by attribute value, you can
+# check that the size of each marker correctly relates to an attribute value by
+# providing the geometry, point_gdf here, and the attribute the size is based
+# off of, which is the 'size' column in this case.
+
+
+
+# Check point geometry location (x, y location)
+vector_test.assert_points(point_gdf)
+
+# Check points are grouped plotted by type
+vector_test.assert_points_grouped_by_type(point_gdf, "size")
+
+# Check points size is relative to a numeric attribute value
+vector_test.assert_collection_sorted_by_markersize(point_gdf, "size")
+
+################################################################################
+# Test Line Attribute Values and Coordinate Information (x, y)
+# ------------------------------------------------------------
+# You can also test the position and plot values of line
+# geometries.
+
+# Check line geometry
+vector_test.assert_lines(multi_line_gdf)
+
+# Check lines plotted by type
+vector_test.assert_lines_grouped_by_type(multi_line_gdf, "attr")
+
+################################################################################
+# Testing Polygon Geometry
+# ------------------------
+# Currently, MatPlotCheck is unable to check that polygons were plotted by type.
+# Eventually this will be supported. For the time being though, you can still
+# check that polygons are plotted correctly!
+
+# Check Polygons
+vector_test.assert_polygons(polygon_gdf)
+
+################################################################################
+# Access the Axes object in a Jupyter Notebook
+# --------------------------------------------
+# MatPlotCheck can be used to help grade Jupyter Notebooks as well. The main
+# difference is in how you would store the Axes from the plot you are grading.
+# Below is an example of how you could store the Axes of a plot you are hoping
+# to grade in a notebook.
+
+# First, import the Notebook module from MatPlotCheck
+import matplotcheck.notebook as nb
+
+# Plot your data
+fig, ax = plt.subplots()
+polygon_gdf.plot(ax=ax, color="purple")
+
+# Plot your line data by attributes using groupby
+for ctype, lines in multi_line_gdf.groupby('attr'):
+    color = line_symb[ctype]
+    label = ctype
+    lines.plot(color=color, ax=ax, label=label)
+
+# Plot your points data by size using groupby
+for ctype, points in point_gdf.groupby('size'):
+    color = point_symb[ctype]
+    label = ctype
+    points.plot(color=color, ax=ax, label=label, markersize=ctype)
+
+# Add a legend
+ax.legend(title="Legend", loc=(1.1, .1));
+
+# HERE'S WHERE YOU STORE THE PLOT!
+# This line at the end of a cell you are expecting a plot in will store any
+# matplotlib plot made in that cell so you can test it at a later time.
```

Comment:
"Here is where you access the axes objects of the plot for testing.
You can add the code line below to the end of any plot cell to store all axes objects created 
by matplotlib in that cell."